ls1intum/Artemis
# `General`: Improve layout of exercise management overview

### Checklist
#### General


#### Client

### Motivation and Context


Improve the layout of the exercise manage over view to use the space more efficiently.

### Description

The course header is now even shorter still including longer descriptions by reducing font size of the description.
Move headlines and create exercise buttons in the exercise overview (see screenshot)

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student

1. Log in to Artemis
2. Navigate to Course Administration
3. Check that the course header looks good with different screen sizes, descriptions lengths and with and without a course icon.
4. Check that the exercise management overview for the instructor looks good and displays all relevant information
5. Check that creating a programming, quiz and modeling/text/fileupload (one of the three) still works

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage





### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/38322605/6b40be49-9a8f-4e8d-bab8-dd01281b98a7)
